Projects/OPS/OPS04/OPS04-AWS200.md

```diff
@@ -0,0 +1,49 @@
+# OPS04-AWS100 - Create a CloudWatch Alarm
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 200 (Intermediate)
+
+## Project's Author(s)
+- [@andrewbrown](https://twitter.com/andrewbrown)
+
+## Objectives
+
+### You need to complete the following:
+- Launch an EC2 t2-micro instance with a public IP address and supply the [provided bash script](OPS04-AWS200-userdata.sh) to install a simple website with an apache server in the UserData.
+- Visit the the public IP so that you are generating NetworkIn. You need
+  to do this so the Metric appears selectable when create your
+CloudWatch Alarm
+- Create a CloudWatch Alarm and use EC2 NetworkIn as the metric
+- Set your CloudWatch Alarm to use a 5 minute period
+- Set your CloudWatch Alarm to a very low static threshold such as 5000
+- Set the Datapoint to alarms to 3 of 4
+- Try to get the Alarm to trigger an Alert state by visiting the website
+  and generating NetworkIN
```

Comment:
The threshold is 5000, so the site needs to be accessed more than 5000 times in 3, 5-minute periods out of 4? Or is the static threshold not 1 http request = 1?

Reply:
@what-name I think its 5000 bytes of data, since that what Network In on EC2 is measured in, please correct me if I am wrong.
@omenking ^^